pkg/app/pipedv1/plugin/kubernetes_multicluster/provider/applier.go

```diff
@@ -157,10 +157,7 @@ func (a *Applier) Delete(ctx context.Context, k ResourceKey) (err error) {
 }
 
 // getNamespaceToRun returns namespace used on kubectl apply/delete commands.
-// priority: config.KubernetesDeploymentInput > kubernetes.ResourceKey
+// Applier.input.Namespace is not used here because it is referenced when the manifest is loaded.
 func (a *Applier) getNamespaceToRun(k ResourceKey) string {
```

Comment:
We might remove the method by using `ResourceKey.Namespace()`. WDYT?
https://github.com/pipe-cd/pipecd/blob/fa5b3b3516912810ab6e2b8043f17712e03e9b69/pkg/app/pipedv1/plugin/kubernetes/provider/resource.go#L55C1-L57C2

Reply:
Oh, I missed it. Thank you.

Reply:
I fixed them to use it.
https://github.com/pipe-cd/pipecd/pull/5865/commits/6012119d5f98a2baeb9f72879c983712eab35a46
https://github.com/pipe-cd/pipecd/pull/5865/commits/5bf3b88fe849fad1ca33bcb8f072eb67c96be752